cw/capture.py

```diff
@@ -1772,17 +1762,10 @@ def capture_ecdsa_sections(ot, fw_bin, pll_frequency, num_sections, secret_k, pr
 
     """
 
-    # This determines if we want to reprogram the firmware
-    # before every sign operation.The default value is false.
-    reset_firmware = False
-
     # Create a temporary buffer to keep the collected sections
     tmp_buffer = np.array([])
     for ii in range(num_sections):
         # Reflash the firmware and set the CW310 pll frequency
```

Comment:
I think also this comment needs to be removed. Similar occurrence below.

Reply:
Agreed, I've removed this comment in both places with the latest change